examples/utils/initializeWebWorkers.js

```diff
@@ -1,7 +1,7 @@
 // This script will load the WebWorkers and Codecs from unpkg url
 try {
   window.cornerstoneWADOImageLoader.webWorkerManager.initialize({
-    maxWebWorkers: 4,
+    maxWebWorkers: 1,
```

Comment:
Does it fail with 4?

Reply:
Fixed - it is ok with 4, just wanted it low for testing so I was only running 1 webworker for debugging.